worksheet_A/TerminalHacking.cpp

```diff
@@ -41,6 +73,52 @@ int main()
 
 	// TODO: implement the rest of the game
 
+	int turnsLeft = 4;
+	string wordGuessed;
+
+	cout << endl;
+	cout << "Please guess the password" << endl;
+
+	while (turnsLeft > 0)
+		{
+			if (turnsLeft > 0)
+			{
+				cout << "Guesses left : " << turnsLeft << endl;
+				cin >> wordGuessed;
+				turnsLeft -= 1;
+			}
+			
+			if (listedWord(wordGuessed, options))
+			{
+				//If words match you win
+				if (wordGuessed == secret)
+				{
+					cout << "You win!" << endl;
+					turnsLeft = 0;
```

Comment:
Needs a comment that this is to break out of the loop. But maybe more readable to use the `break` keyword